prov/tcp/src/tcpx_shared_ctx.c

```diff
@@ -98,6 +98,7 @@ static ssize_t tcpx_srx_recvmsg(struct fid_ep *ep, const struct fi_msg *msg,
 		goto unlock;
 	}
 
+	flags |= srx_ctx->op_flags &  FI_MULTI_RECV;
```

Comment:
Why is this modifying the user flags by potentially OR'ing in the mult_recv flag?

Reply:
My understanding is that if srx_ctx is created with multi_recv flag set in rx_attr->op_flags, then all the buffers posted to that srx queue should be treated as multi recv buffers. So here I am checking the op_flags and setting it in flags for this posted buffer.

Reply:
"Operation flags define the default flags applied to an endpoint’s data transfer operations, where a flags parameter is not available."  The 'msg' calls allow the app to override the default flags.  Without this ability, the flags could not easily be cleared.

Reply:
Got it. msg flags override attr flags, not the other way.